dlt/helpers/streamlit_app/menu.py

```diff
@@ -0,0 +1,42 @@
+import dlt
+import streamlit as st
+
+from dlt.common.destination.reference import WithStateSync
+from dlt.helpers.streamlit_app.utils import HERE
+from dlt.helpers.streamlit_app.widgets import logo, stat, tag, mode_selector
+from dlt.helpers.streamlit_app.blocks.load_info import last_load_info
+from dlt.pipeline.state_sync import load_pipeline_state_from_destination
+
+
+def menu(pipeline: dlt.Pipeline) -> None:
+    mode_selector()
+    logo()
+    st.page_link(f"{HERE}/dashboard.py", label="Explore data", icon="🕹️")
```

Comment:
does that mean that each page is a separate script with separate main function? I think streamlit is a single page app by nature. I'm not against changing the structure but your change added code duplication and will result in more bugs later

Reply:
yes each page has it's own main function in our case but we can also remove them this is however will not prevernt streamlit from complete re-renderings.

Reply:
So now I created an `index.py` which redirects to dashboard when first loaded.